pom.xml

```diff
@@ -59,7 +59,7 @@
     make sure the resulting binary tarball contains only
     required jars, and that no jar has an offending license.
     -->
-    <driver.version>4.14.1</driver.version>
+    <driver.version>4.16.0</driver.version>
```

Comment:
[Optional] It _may_ be better to incorporate [this Java Driver enhancements](https://github.com/datastax/java-driver/pull/1641)

Reply:
it is definitely required to use 4.16.0 and I would need to have 4.16.1 with this patch
https://github.com/datastax/java-driver/pull/1643

Reply:
Re: question from @msmygit ... we can move to a Java driver that includes the changes in the PR you mention but they certainly aren't essential for what we're doing here.  It would be a bit nicer to not have to create lots of builder objects for every vector we parse from input but that's a relatively small cost to pay; the JVM is very good at creating (and reaping) lots of small objects. :)

Re: question from @eolivelli .... I agree that 4.16.0 is absolutely required for this change.  The constraint on anything beyond 4.16.0 refers to [another PR](https://github.com/datastax/dsbulk/pull/476) along similar lines.  This PR uses 4.16.0 as it stands.